api/debug/flags.go

```diff
@@ -49,68 +49,118 @@ var (
 	}
 	pprofFlag = cli.BoolFlag{
 		Name:  "pprof",
 		Usage: "Enable the pprof HTTP server",
 	}
 	pprofPortFlag = cli.IntFlag{
-		Name:  "pprofport",
+		Name:  "pprof.port",
 		Usage: "pprof HTTP server listening port",
 		Value: 6060,
 	}
 	pprofAddrFlag = cli.StringFlag{
-		Name:  "pprofaddr",
+		Name:  "pprof.addr",
 		Usage: "pprof HTTP server listening interface",
 		Value: "127.0.0.1",
 	}
 	memprofileFlag = cli.StringFlag{
-		Name:  "memprofile",
+		Name:  "pprof.memprofile",
 		Usage: "Write memory profile to the given file",
 	}
 	memprofilerateFlag = cli.IntFlag{
-		Name:  "memprofilerate",
+		Name:  "pprof.memprofilerate",
 		Usage: "Turn on memory profiling with the given rate",
 		Value: runtime.MemProfileRate,
 	}
 	blockprofilerateFlag = cli.IntFlag{
-		Name:  "blockprofilerate",
+		Name:  "pprof.blockprofilerate",
 		Usage: "Turn on block profiling with the given rate",
 	}
 	cpuprofileFlag = cli.StringFlag{
-		Name:  "cpuprofile",
+		Name:  "pprof.cpuprofile",
 		Usage: "Write CPU profile to the given file",
 	}
 	traceFlag = cli.StringFlag{
 		Name:  "trace",
 		Usage: "Write execution trace to the given file",
 	}
+	// (Deprecated June 2021)
+	legacyPprofPortFlag = cli.IntFlag{
+		Name:  "pprofport",
+		Usage: "pprof HTTP server listening port (deprecated, use --pprof.port)",
+		Value: 6060,
+	}
+	legacyPprofAddrFlag = cli.StringFlag{
+		Name:  "pprofaddr",
+		Usage: "pprof HTTP server listening interface (deprecated, use --pprof.addr)",
+		Value: "127.0.0.1",
+	}
+	legacyMemprofileFlag = cli.StringFlag{
+		Name:  "memprofile",
+		Usage: "Write memory profile to the given file (deprecated, use --pprof.memprofile)",
+	}
+	legacyMemprofilerateFlag = cli.IntFlag{
+		Name:  "memprofilerate",
+		Usage: "Turn on memory profiling with the given rate (deprecated, use --pprof.memprofilerate)",
+		Value: runtime.MemProfileRate,
+	}
+	legacyBlockprofilerateFlag = cli.IntFlag{
+		Name:  "blockprofilerate",
+		Usage: "Turn on block profiling with the given rate (deprecated, use --pprof.blockprofilerate)",
+	}
+	legacyCpuprofileFlag = cli.StringFlag{
+		Name:  "cpuprofile",
+		Usage: "Write CPU profile to the given file (deprecated, use --pprof.cpuprofile)",
+	}
+	legacyBacktraceAtFlag = cli.StringFlag{
+		Name:  "backtrace",
+		Usage: "Request a stack trace at a specific logging statement (e.g. \"block.go:271\") (deprecated, use --log.backtrace)",
+		Value: "",
+	}
+	legacyDebugFlag = cli.BoolFlag{
+		Name:  "debug",
+		Usage: "Prepends log messages with call-site location (file and line number) (deprecated, use --log.debug)",
+	}
 )
 
 // Flags holds all command-line flags required for debugging.
 var Flags = []cli.Flag{
 	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag,
-	pprofFlag, pprofAddrFlag, pprofPortFlag,
+	logjsonFlag, pprofFlag, pprofAddrFlag, pprofPortFlag,
 	memprofileFlag, memprofilerateFlag,
 	blockprofilerateFlag, cpuprofileFlag, traceFlag,
 }
```

Comment:
It would be better if `DeprecatedFlags` to `Flags` are defined in the same place. The two type flags should be able to used at the same place.
Option 1) Adding `DeprecatedFlags` to `Flags`
```
var Flags = append([]cli.Flag{
	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag,
	logjsonFlag, pprofFlag, pprofAddrFlag, pprofPortFlag,
	memprofileFlag, memprofilerateFlag,
	blockprofilerateFlag, cpuprofileFlag, traceFlag,
}, DeprecatedFlags...)
```
Option 2) Just adding all flags together with comments
```
var Flags = []cli.Flag{
	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag,
	logjsonFlag, pprofFlag, pprofAddrFlag, pprofPortFlag,
	memprofileFlag, memprofilerateFlag,
	blockprofilerateFlag, cpuprofileFlag, traceFlag,
       // Depcrecated
      	legacyPprofPortFlag,
	legacyPprofAddrFlag,
	legacyMemprofileFlag,
	legacyMemprofilerateFlag,
	legacyBlockprofilerateFlag,
	legacyCpuprofileFlag,
	legacyBacktraceAtFlag,
	legacyDebugFlag,
}
```

Reply:
I'll add show-deprecated-flags command in next [PR](https://github.com/hyochan-brown/klaytn/commit/3e13156c7a70c59ec84bce08a9ae4b323ddd3f0d) for ethereum compatibility(After all flag PR merge).
I intend to bring together all deprecated flags(debug, rpc, ... flags) in same place.

Reply:
I don't think placing the deprecated flags in the same place is a good idea.
1) We won't be able to delete those deprecated flags. No one knows which one is being used.
2) All flags are not called at the same place. For example, some flags are only used at `start` command while others are used at `start` and `init`.

Reply:
Winnie! 
Thanks good review!
1.  if it becomes the version to be deprecated, I think we can delete it regardless of whether it is used or not. 
2. Is there a reason why where it is used and where it is declared should be the same...?

Reply:
cc. @winnie-byun

Reply:
I reflected it.
Thanks winnie!